src/extension.ts

```diff
@@ -65,6 +65,8 @@ export async function activate(context: vscode.ExtensionContext) {
         vscode.commands.registerCommand('clusters.openshift.deploy.dc.showLog.palette', (context) => execute(DeploymentConfig.showLog, context)),
         vscode.commands.registerCommand('clusters.openshift.deploy.delete', (context) => execute(DeploymentConfig.delete, context)),
         vscode.commands.registerCommand('clusters.openshift.deploy.delete.palette', (context) => execute(DeploymentConfig.delete, context)),
+        vscode.commands.registerCommand('clusters.openshift.deploy.rc.showLog', (context) => execute(DeploymentConfig.rcShowLog, context)),
```

Comment:
ids should be rcShowLog and rcShowLog.palette or tests will try to run actual commands.